<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SL-1155.attlocal.net" failures="3" tests="7" name="Command line test" time="39.375" errors="0" timestamp="6 Feb 2019 23:38:23 GMT">
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="22.865">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at com.DemoTest.Tests.WebDemoTest.LoginTest(WebDemoTest.java:40)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="25.78">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at com.DemoTest.Tests.WebDemoTest.LoginTest(WebDemoTest.java:40)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="26.091"/>
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="31.832"/>
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="33.706"/>
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="34.627"/>
  <testcase classname="com.DemoTest.Tests.WebDemoTest" name="LoginTest" time="38.98">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at com.DemoTest.Tests.WebDemoTest.LoginTest(WebDemoTest.java:40)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Command line test -->
